package.json

```diff
@@ -6,7 +6,7 @@
   "main": "src/microcosm.js",
   "scripts": {
     "build": "make all",
-    "lint": "eslint {src,test,examples}/**/*.{js,jsx}",
+    "lint": "eslint {src,test,examples}/**",
```

Comment:
very odd. eslint version difference perhaps? guess that would only happen if you were somehow using a global copy.